code/modules/antagonists/blob/blob_mobs.dm

```diff
@@ -128,6 +128,8 @@
 	var/is_zombie = FALSE
 	///Whether or not this is a fragile spore from Distributed Neurons
 	var/is_weak = FALSE
+	///Whether the spore requires a factory to survive - currently used by Distributed Neurons reagent zombies
+	var/needs_factory = TRUE
```

Comment:
do the blob spores from cytology have this set to FALSE?

Reply:
No, but they do apparently have a different var that I missed and that can more or less fulfil the same purpose. So, thanks for pointing it out, I'll use that instead.